ci-operator/config/openshift/jenkins/openshift-jenkins-release-4.13__jenkins-hypershift4.13.yaml

```
@@ -0,0 +1,71 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  ocp_builder_rhel-8-golang-1.19-openshift-4.13:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.19-openshift-4.13
+  upi-installer:
+    name: "4.13"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  from_repository: true
+images:
+- dockerfile_literal: |-
+    FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.13
+    COPY . .
+    RUN chgrp -R 0 /go && \
+    chmod -R g=u /go
+    RUN mkdir -p /.cache
+    RUN chgrp -R 0 /.cache && \
+    chmod -R g=u /.cache
+  to: jenkins-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: aws-jenkins-lp-rosa-hypershift
+  cron: 0 6 * * 1
```

```suggestion
  cron: 0 10 * * 1
```
Per this message https://redhat-internal.slack.com/archives/C04PK4QPSR1/p1687889369616619